exploratory_analysis.ipynb

```diff
@@ -1913,125 +1926,136 @@
+       "      <td>1523</td>\n",
+       "      <td>1523</td>\n",
+       "      <td>1523</td>\n",
+       "      <td>1523</td>\n",
+       "      <td>1523</td>\n",
+       "      <td>1523</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "    state     measure_name  state_code  county_code  measure_id  numerator  \\\n",
-       "519    TX  Premature Death     78096.0     412057.0      1627.0  1621392.0   \n",
-       "124    GA  Premature Death     14391.0     178255.0      1107.0   736457.0   \n",
-       "207    KY  Premature Death     17598.0     100398.0       838.0   407621.0   \n",
-       "543    VA  Premature Death     47685.0     248522.0       935.0   553747.0   \n",
-       "291    MO  Premature Death     23113.0      93377.0       797.0   496435.0   \n",
+       "                      measure_name state  county  state_code  county_code  \\\n",
+       "499                   Unemployment    TX    1778        1778         1778   \n",
+       "94             Children in poverty    TX    1778        1778         1778   \n",
+       "346                Premature Death    TX    1627        1627         1627   \n",
+       "142  Daily fine particulate matter    TX    1524        1524         1524   \n",
+       "601             Violent crime rate    TX    1523        1523         1523   \n",
        "\n",
-       "     denominator     raw_value   fips_code  start_year  \n",
-       "519          0.0  1.361825e+07  78508057.0   3262320.0  \n",
-       "124          0.0  1.072616e+07  14569255.0   2219550.0  \n",
-       "207          0.0  8.137653e+06  17698398.0   1680195.0  \n",
-       "543          0.0  7.667041e+06  47933522.0   1874682.0  \n",
-       "291          0.0  6.922232e+06  23206377.0   1598005.0  "
+       "     measure_id  numerator  denominator  raw_value  fips_code  start_year  \n",
+       "499        1778       1778         1778       1778       1778        1778  \n",
+       "94         1778       1778            0       1778       1778        1778  \n",
+       "346        1627       1627            0       1627       1627        1627  \n",
+       "142        1524          0            0       1524       1524        1524  \n",
+       "601        1523       1523         1523       1523       1523        1523  "
       ]
      },
-     "execution_count": 227,
+     "execution_count": 259,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "# Top five states with the higest raw_value in group 2\n",
+    "# Top five measures with the most occurance in different states in group 2\n",
     "\n",
-    "df2_state = df2.groupby(['state', 'measure_name']).sum().reset_index()\n",
-    "df2_state.sort_values('raw_value', ascending=False).head()"
+    "df2_state = df2.groupby(['measure_name','state']).count().reset_index()\n",
+    "d2 = df2_state.sort_values('raw_value', ascending=False).head()\n",
```

Comment:
Ahh, I see you did the same here with a new df named "d2" so I guess that is what the intent is, just checking